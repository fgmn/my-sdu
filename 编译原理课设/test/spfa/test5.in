100
500
0
24
27
0
70
24
0
19
1
0
83
30
0
65
3
0
34
24
0
74
1
0
45
13
1
15
18
1
81
25
1
6
1
1
98
13
1
33
27
1
2
21
1
26
20
1
29
20
1
20
4
1
45
16
2
80
3
2
66
26
2
48
29
2
5
17
2
71
29
2
87
27
2
86
12
2
41
13
3
43
1
3
87
2
3
74
19
3
3
14
3
69
16
3
56
7
3
45
18
4
87
25
4
49
17
4
80
29
4
13
16
4
34
5
4
83
16
4
96
17
4
78
9
4
63
22
4
8
22
5
97
9
5
62
18
5
87
16
5
94
9
5
17
15
5
92
7
5
90
6
5
93
26
5
92
25
6
35
25
6
53
19
6
99
2
6
44
15
6
29
1
6
44
14
6
8
25
6
54
19
6
92
8
6
98
12
6
25
16
7
16
24
7
82
12
7
46
10
7
78
7
7
27
24
7
48
7
7
72
2
7
63
3
7
67
13
8
19
19
8
85
14
8
91
3
8
24
9
8
73
20
8
60
22
8
93
19
9
69
13
9
32
14
9
53
9
9
33
16
9
68
12
9
11
8
10
36
24
10
68
22
10
87
16
10
29
8
10
78
22
10
78
12
10
45
25
10
26
30
10
83
10
10
24
8
10
39
9
10
95
6
11
66
19
11
55
1
11
36
18
11
37
28
11
89
12
11
30
8
11
86
28
11
34
20
11
65
12
12
76
21
12
80
14
12
53
18
12
63
10
12
80
13
12
82
19
13
71
14
13
44
15
13
58
21
13
84
19
14
66
30
14
74
9
14
29
7
14
19
25
14
31
29
14
42
14
14
71
21
14
18
23
15
78
19
15
63
23
15
15
29
15
91
8
15
99
18
15
60
23
16
53
3
16
65
27
16
89
22
16
57
25
16
38
12
16
83
11
16
95
20
17
24
10
17
80
21
17
17
26
17
52
23
17
78
23
17
24
21
17
95
14
17
40
21
17
53
27
18
78
28
18
67
11
18
32
12
18
41
14
18
53
5
19
27
11
19
87
26
19
73
20
19
32
10
19
62
11
19
61
6
19
54
3
19
55
17
19
41
22
19
31
10
20
32
26
20
52
1
20
61
19
20
39
18
20
77
13
20
71
8
20
41
17
20
42
18
20
76
16
20
60
13
20
61
7
20
50
23
20
41
28
21
32
27
21
96
13
21
43
17
21
38
6
22
41
4
22
68
26
22
79
14
22
22
30
22
28
28
22
84
22
22
65
15
22
71
23
22
49
22
23
32
3
23
57
12
23
55
26
23
45
28
23
24
21
23
91
19
24
54
28
24
76
6
25
86
20
25
76
13
25
34
13
25
28
9
25
82
29
25
26
15
25
82
16
25
67
2
25
81
4
25
30
3
26
97
14
26
33
18
26
61
17
26
56
1
27
97
11
27
50
26
27
42
1
27
84
19
27
40
5
27
65
4
27
84
18
27
79
28
27
57
20
27
88
2
27
71
11
28
40
28
28
55
27
28
56
23
28
64
14
28
51
8
28
43
16
28
78
10
28
41
30
28
37
28
28
40
12
29
87
23
29
52
17
29
83
7
29
49
26
30
52
17
30
53
14
30
61
25
30
50
13
30
72
12
30
66
5
30
87
5
30
46
10
30
86
13
30
57
6
30
40
16
31
63
2
31
89
20
31
54
8
31
96
14
31
35
2
31
92
14
32
42
22
32
39
5
32
73
12
32
78
29
32
84
1
32
95
13
32
96
12
32
63
26
32
42
30
32
64
10
32
65
22
32
45
8
33
87
13
33
35
8
33
96
13
33
92
7
33
61
19
33
89
3
33
52
14
34
73
14
34
66
2
34
46
13
34
55
3
34
36
30
34
91
21
35
48
12
35
76
6
35
77
23
35
94
3
35
95
1
35
75
3
35
37
12
35
87
3
36
68
23
36
60
18
36
43
5
36
51
6
36
67
6
36
43
29
36
89
15
36
78
29
37
73
2
37
75
16
37
91
30
37
60
6
37
75
17
37
67
28
37
78
12
37
38
21
37
79
18
38
43
16
38
48
28
38
78
14
38
53
23
38
94
3
38
92
22
38
96
17
38
91
9
39
75
7
39
48
16
39
98
26
39
55
28
39
55
28
39
52
17
39
47
18
39
79
26
39
40
23
40
46
14
40
98
6
40
83
16
40
46
20
40
99
27
40
89
16
40
69
11
40
86
16
41
70
30
41
88
8
41
73
15
41
77
25
42
59
20
42
69
6
42
69
21
43
91
3
43
71
14
43
80
29
43
53
18
43
60
7
43
99
30
44
78
16
44
71
30
44
91
25
44
54
9
44
84
28
45
91
9
45
78
18
45
97
5
45
83
4
46
88
23
46
86
29
46
77
9
46
78
2
46
63
27
46
60
12
46
88
21
46
62
28
47
62
18
47
73
6
47
75
1
47
84
4
47
94
22
47
96
24
48
62
10
48
60
6
48
71
11
49
81
11
49
63
18
50
96
26
50
60
23
50
50
23
50
80
2
50
86
6
51
70
17
51
86
30
52
60
9
52
93
9
52
99
19
52
64
19
52
63
23
52
96
2
52
63
11
52
55
2
53
89
23
53
91
4
53
79
25
53
59
14
53
74
10
53
92
22
54
57
13
54
79
1
54
89
23
54
68
18
56
74
21
56
99
19
56
87
10
56
93
1
56
66
20
56
63
5
56
92
2
56
89
28
57
62
22
57
60
10
57
76
4
57
93
6
58
59
13
58
71
27
58
88
20
58
62
7
59
84
9
59
64
20
59
71
2
59
86
1
60
89
13
60
65
9
60
87
28
60
74
17
60
80
13
61
62
9
61
67
14
61
71
5
61
87
7
61
87
4
61
72
5
61
73
29
61
74
11
62
82
11
62
84
11
63
96
10
63
95
4
63
64
1
64
68
18
64
72
10
64
83
18
64
99
12
65
95
30
66
97
4
66
96
27
66
89
2
67
98
8
68
74
19
68
83
19
68
88
24
68
90
25
68
98
13
68
86
3
68
97
11
70
91
4
70
83
17
70
86
3
70
90
14
71
87
24
71
90
20
71
74
11
71
91
9
71
93
13
71
81
5
72
92
15
72
91
30
73
75
10
73
79
5
73
81
10
74
96
24
74
86
22
75
82
18
75
90
3
76
90
12
76
94
9
76
84
3
78
90
12
78
99
2
78
97
27
78
86
27
78
98
24
79
80
18
79
98
27
79
91
18
80
92
30
80
80
1
80
87
19
81
97
30
81
84
22
82
94
5
83
93
18
84
89
6
84
91
26
85
91
23
85
94
14
85
91
14
86
92
7
86
89
9
88
99
25
88
95
3
88
90
21
89
97
16
89
98
12
89
96
24
90
94
3
93
99
12
95
97
17
96
96
4
