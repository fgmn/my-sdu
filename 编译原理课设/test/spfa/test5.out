0
590558003
590558003
590558003
590558003
590558003
590558003
590558003
590558003
590558003
590558003
590558003
590558003
590558003
590558003
590558003
590558003
590558003
590558003
1
590558003
590558003
590558003
590558003
27
590558003
590558003
12
590558003
590558003
590558003
11
11
590558003
24
13
54
25
46
16
17
23
13
59
590558003
13
31
34
25
590558003
38
60
33
69
4
18
590558003
17
590558003
33
27
7
12
13
14
3
26
21
22
19
24
12
12
21
1
16
19
36
31
5
23
17
23
17
12
590558003
23
11
14
18
19
21
25
23
16
14
23
18
23
26
